Automattic/woocommerce-payments
# Add filter to record wcpay server api response time

Related PR Automattic/woopay/pull/1396
Fixes Automattic/woopay/issues/1378

#### Changes proposed in this Pull Request
Add filter `wcpay_api_request_response` to record wcpay server api response time 

#### Testing instructions
Test with WooPay PR Automattic/woopay/pull/1396 and make sure nothing is broken

-------------------


**Post merge**
